sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -383,6 +387,7 @@ private void internalExtractColumnDefinitions(@Nullable ColumnIdent columnIdent,
             boolean nullable = !notNullColumns.contains(newIdent);
             columnProperties = furtherColumnProperties(columnProperties);
             Reference.IndexType columnIndexType = getColumnIndexType(columnProperties);
+            boolean columnsStoreDisabled = !(boolean) columnProperties.getOrDefault(DOC_VALUES, true);
```

Comment:
Just curious, why not

`boolean columnsStoreDisabled = columnProperties.getOrDefault(DOC_VALUES, false);`

?

Reply:
columnProperties are defined as Map<String, Object>

Reply:
And how about just 

```
boolean columnsStoreDisabled = (boolean) columnProperties.getOrDefault(DOC_VALUES, false); 
```

?

Reply:
the value will be `false` if doc_values are enabled, default is `true`. so I'll have to negate the result.